contrib/scripts/extract_authors.sh

```diff
@@ -14,4 +14,4 @@ function extract_authors() {
 	done
 }
 
-extract_authors | uniq | sort
+extract_authors | sort -u
```

Comment:
Since you edited this file, can you also add this on top and generate the `AUTHORS` file again?
```
# Ensure sort order doesn't depend on locale
export LANG=C
export LC_ALL=C
```

Reply:
Would `LANG=en_US.UTF-8` we reasonable?